moto/glue/constants.py

```diff
@@ -0,0 +1,33 @@
+import re
+
+# This file contains the constants required for Glue Schema Registry APIs.
```

Comment:
Do you want to rename the file indicate that these constants are only for Glue Schema Registry APIs?